Automattic/wp-calypso
# WordPressLogo viewbox clipping

#### Proposed Changes

* Updates the `viewBox` for `WordPressLogo` to fit the entire logo.
* This logo is used in the /start header it is separate from the `<Girdicon icon="my-sites"/>` used elsewhere.

#### Testing Instructions

* Compare http://calypso.localhost:3000/start/user and https://wordpress.com/start/user
* The logo should not be clipped

#### Before
![Screenshot 2022-06-10 at 16-07-25 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/173002417-f7c40d85-f876-4640-9257-6792d142ddc9.png)


#### After
![Screenshot 2022-06-10 at 16-15-45 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/173002473-538da41b-e786-4d4c-b932-6a1313a1ccd3.png)


Related to #
